cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -130,21 +127,17 @@ public static CruiseControl fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup ver
                     kafkaAssembly.getMetadata().getName(),
                     Labels.fromResource(kafkaAssembly).withKind(kafkaAssembly.getKind()));
 
-        CruiseControlSpec spec  = kafkaAssembly.getSpec().getCruiseControl();
+        KafkaSpec kafkaSpec  = kafkaAssembly.getSpec();
+        CruiseControlSpec spec  = kafkaSpec.getCruiseControl();
 
         if (spec != null) {
             cruiseControl.isDeployed = true;
 
-            cruiseControl.setReplicas(spec != null && spec.getReplicas() > 0 ? spec.getReplicas() : DEFAULT_REPLICAS);
+            cruiseControl.setReplicas(spec.getReplicas() > -1 ? spec.getReplicas() : DEFAULT_REPLICAS);
```

Comment:
So I think we should just set something like 

`@JsonProperty(required = true)`

on the replicas in the Cruise Control spec class, so that you don't need this check and `DEFAULT_REPLICAS` as well.
You can check how it behaves in `KafkaClusterSpec` and `KafkaCluster` classes.
Wdyt?